spec/features/hpv_vaccination_administered_spec.rb

```diff
@@ -72,6 +72,21 @@
     then_i_see_a_success_message
   end
 
+  scenario "Administered without registration" do
```

Comment:
@thomasleese I opted to put the test here so I can reuse a good number of steps. What do you think? Still go ahead and create a separate file? I don't mind either way.

Reply:
I think this works!